lib/commands.sh

```
@@ -328,64 +366,412 @@ cmd_policy() {
+
+  if [ -f "$policy_file" ]; then
+    case "$strictness" in
+      open)
+        if ! jq '(.version // 1) as $v | .version=$v | .require_signed=false' "$policy_file" >"$tmp" 2>/dev/null; then
+          rm -f "$tmp"; die "shiplog: failed to update $policy_file"
+        fi
+        ;;
+      balanced)
+        # Build authors JSON array from input
+        local authors_json
+        authors_json=$(printf '%s\n' $authors_input | jq -R -s 'split("\n") | map(select(length>0)) | unique')
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)